### PR TITLE
Fix the github test to match new github api.

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: tests/validation/cattlevalidationtest/core/test_github.py [hunk 1]
```diff
@@ -60,13 +60,16 @@ def config():
 @pytest.fixture(scope='module')
 def github_request_code(config, cattle_url, admin_client, request, user=None):
     def fin():
-            admin_client.create_githubconfig(enabled=False)
+            admin_client.create_githubconfig(enabled=False,
+                                             accessMode='restricted')
```
Comment (alena1108) [inline tests/validation/cattlevalidationtest/core/test_github.py]: James, does this change mean that there is no default value for accessMode? As before we weren't passing anything for this param.

Comment (hibooboo2) [inline tests/validation/cattlevalidationtest/core/test_github.py]: There is still a default. We just should leave it as restricted after test cases.

**Comment by cjellick** [comment]
LGTM

**Comment by alena1108** [comment]
LGTM

**Pull request merged**